sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/util/PassThroughWindowFn.java

```diff
@@ -30,24 +33,32 @@
 /**
  * A {@link WindowFn} that leaves all associations between elements and windows unchanged.
  *
- * <p>In order to implement all the abstract methods of {@link WindowFn}, this requires the
- * prior {@link WindowFn}, to which all auxiliary functionality is delegated.
+ * <p>This {@link WindowFn} is applied when elements must be passed through a {@link GroupByKey},
```

Comment:
May also want to note that one of the big reasons for this is that a merging window Fn will have its "merginess" removed after group by key. This means it can't be used with another group by key without either (1) reinstating its merginess or (2) switching to a non-merging version like this.